### PR TITLE
TAJO-1380: Update JDBC documentation for new JDBC driver

### PR DESCRIPTION
**Pull request by dongjoon-hyun**

### CODE REVIEW

File: tajo-docs/src/main/sphinx/jdbc_driver.rst [hunk 1]
```diff
@@ -9,11 +9,15 @@ In this section, we explain how to get JDBC driver and an example code.
 How to get JDBC driver
 =======================
 
+From Tajo Homepage
```
Comment (hyunsik) [inline tajo-docs/src/main/sphinx/jdbc_driver.rst]: I have one trivial suggestion. How about 'Direct Download'? In my view, It would be better.

**Comment by hyunsik** [comment]
Thank you for your work. The patch looks good to me. I leaved one trivial comment.

**Comment by dongjoon-hyun** [comment]
Thank you for your feedback. No problem. Do you mean the following links? Which one do you prefer?

1. http://apache.org/dyn/closer.cgi/tajo/tajo-0.10.0/tajo-jdbc-0.10.0.jar

2. http://www.us.apache.org/dist/tajo/tajo-0.10.0/tajo-jdbc-0.10.0.jar

By the way, those have a version number in the link, so we should update carefully.

**Comment by hyunsik** [comment]
I just meant the title name. The link 'http://tajo.apache.org/downloads.html' would be best because this page can track the number of downloads for each release.

**Comment by dongjoon-hyun** [comment]
Oh, I see. You mean literally 'Direct Download' instead of 'From Tajo Homepage'.
Okay, I will change soon.

**Comment by hyunsik** [comment]
Thank you :)

**Comment by dongjoon-hyun** [comment]
Oh, I updated my pull request here (GitHub) and forgot to upload in Jira issue. I will update the attachement file in Jira issue soon.

**Pull request closed without merging**